charts/sysdig/values.yaml

```diff
@@ -26,11 +26,11 @@ resources:
   # a sane defaults ones. If you have more questions or concerns, please refer
   # to Sysdig Support for more info about it
   requests:
-    cpu: 600m
-    memory: 512Mi
+    cpu: 3000m
```

Comment:
This can't be the default value. I might be missing something, but it's way above our advertised number.

Reply:
It matches our "mid sized" node config (~32 core nodes on ~100 node clusters)